src/main/java/org/repodriller/scm/SCM.java

```diff
@@ -31,6 +31,7 @@
 	long totalCommits();
 	void reset();
 	void checkout(String id);
+	void getDiffBetweenCommits(String priorCommit, String laterCommit);
```

Comment:
Why does this return void? Where is the diff stored?

Reply:
Temporary. I meant to ask some questions about the format in which we want this information, but you're too quick for me :)